### ISSUE TITLE
Step parsing pattern cannot handle empty parameters

### ISSUE THREAD

**Issue by dimovelev** [witspirit/IntelliJBehave#94]
I can define a step like that:
```
@Given("a user$name")
@Alias("user $name")
public void givenUser(String name) {
}
```
In the @Given pattern there is no space before $name on purpose - this works with jbehave and it would call the step with an empty name.
The IDEA plugin does not seem to match this and shows the step in the story as unknown.

**Comment by picimako** [comment]
@dimovelev 
I have other things on my plate at the moment, but I'll try to look into it soon (at least to see what the problem might be), and if I find the cause, I'll most probably be able to work on it around Christmas time.

**Comment by picimako** [comment]
@dimovelev 
A fix for this is published and available in v1.67.2 Let me know if it fixed it for you too.
